packages/core/src/components/graphs/wordcloud.ts

```diff
@@ -51,18 +51,27 @@ export class WordCloud extends Component {
 			return;
 		}
 
+		const wordCloudData = [];
+		displayData.map(function (d) {
+			let value = d[fontSizeMapsTo];
+
+			if (typeof d[fontSizeMapsTo] !== 'number') {
+				throw Error(
+					'Badly formatted WordCloud data. `value` should only be an integer or float'
+				);
+			}
+
+			wordCloudData.push({
+				[groupMapsTo]: d[groupMapsTo],
+				text: d[wordMapsTo],
+				size: value,
+				value,
+			});
+		});
+
```

Comment:
```suggestion

```

Reply:
If this is the approach we have to take since some words are still rendering and then the error message is being shown, then instead of pushing, we should just be returning the object in the map function & `worldCloudData = displayData.map(function (d)....`

The current approach makes it seem that `map` is being used as a forEach